tests/src/FormDataHandler.php

```diff
@@ -0,0 +1,39 @@
+<?php
+require_once('../../vendor/autoload.php');
```

Comment:
Shouldn't need to load the autoloader in the tests, just manually require the files

Reply:
Removed.